tests/e2e/pickdiff.spec.js

```diff
@@ -0,0 +1,115 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
+
+test.describe('PickDiff Application', () => {
+  test('should load the main page successfully', async ({ page }) => {
+    await page.goto('/');
+    
+    // Check that the page title is correct
+    await expect(page).toHaveTitle('PickDiff');
+    
+    // Check for main elements
+    await expect(page.locator('h1')).toContainText('PickDiff');
+    await expect(page.locator('#repo-path-container')).toBeVisible();
+    await expect(page.locator('#start-commit')).toBeVisible();
+    await expect(page.locator('#end-commit')).toBeVisible();
+    await expect(page.locator('#file-tree')).toBeVisible();
+  });
+
+  test('should display repository path', async ({ page }) => {
+    await page.goto('/');
+    
+    // Wait for repository path to load
+    await expect(page.locator('#repo-path')).not.toBeEmpty();
+    const repoPath = await page.locator('#repo-path').textContent();
+    expect(repoPath).toBeTruthy();
```

Comment:
We probably can't test for an absolute path here since the repo path will be different when running on different machines (e.g. CI vs local development), but would it be better to at least verify the repo name is in the path?